### DISCUSSION TITLE
Uploading a photo

### DISCUSSION THREAD

**Discussion by controllerandmouse** [community/community#65715] [New to GitHub]
Hello, 

I recently created a github account and I would like to upload a custom profile photo but I'm having some troubles finding it.

Thanks!

**Comment by jurecapuder** [comment]
Hello,

Welcome to GitHub! I'd be happy to assist you with uploading a custom profile photo to your GitHub account. Follow these steps to set up your custom profile picture:

1. **Sign in to GitHub**: Visit the GitHub website (https://github.com/) and sign in to your GitHub account if you're not already logged in.

2. **Access Your Profile**: Click on your profile picture or your username in the top right corner of the GitHub homepage to access your profile.

3. **Edit Profile**: On your profile page, you'll see a green button that says "Edit profile" just below your profile picture. Click on it.

4. **Upload a Picture**: Scroll down to the "Profile Picture" section. Here, you can either drag and drop an image file from your computer or click the "Upload new picture" button to select an image from your local storage.

5. **Crop and Adjust**: Once you've uploaded your image, you may need to crop it to fit the profile picture dimensions. Follow the on-screen instructions to adjust and crop your image as needed.

6. **Save Changes**: After cropping and making any necessary adjustments, click the "Set new profile picture" button to save your changes.

7. **Confirm Gravatar**: GitHub may also ask you to confirm your Gravatar email address. If prompted, follow the instructions to confirm your Gravatar email, as this can be an alternative way to set your profile picture.

8. **Preview and Confirm**: Review how your new profile picture looks on the profile page. If you're satisfied with the image and how it appears, click the "Update profile" button to confirm your changes.

That's it! Your custom profile picture should now be set and visible on your GitHub profile. It may take a moment for the changes to fully propagate.

Please ensure that your profile picture adheres to GitHub's community guidelines, which generally require it to be a recognizable image of yourself or a suitable representation.

**Comment by Raoufkorishkh** [comment]
![1687330427877](https://github.com/community/community/assets/135653797/8272bb77-f6e2-4c3b-9134-b7065f6eeadc)

**Comment by Raoufkorishkh** [comment]
Raouf khan is currently in this situation.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
